File: frontend/src/data/Conflict.ts
import { Handyman } from "./Handyman";
import { Offer } from "./Offer";
import { StatusOfTask } from "./StatusOfTask";
import { User } from "./User";

export interface Conflict {
    id: number;
    clientId: number;
    handymanId: number;
    offerId: number;
    description?: string;
    status: StatusOfTask;
    created?: Date;
    client: User;
    offer: Offer;
    handymen: Handyman;
  }

File: backend/Entities/Conflict.cs
using Handy.Data.Enum;
using Handy.Entities;

namespace Handy.Api.Entities
{
    public class Conflict
    {
        public int Id { get; set; }
        public int ClientId { get; set; }
        public User Client { get; set; }
        public int HandymanId { get; set; }
        public int OfferId { get; set; }
        public Offer Offer { get; set; }
        public string? Description { get; set; }
        public StatusOfTask Status {  get; set; }
        public DateTime? Created { get; set; } = DateTime.Now;
    }
}
